ethergo/chain/chain.go

```diff
@@ -39,6 +39,8 @@ type Chain interface {
 	Estimator() gas.PriceEstimator
 	// GasSetter gets the gas setter
 	GasSetter() gas.Setter
+	// ChainConfig gets the chain config.
+	ChainConfig() *params.ChainConfig
```

Comment:
if you want you can remove this, but probably good to have. As an aside, we've really gotta deprecate this entire chain package soon.